ex2_services/Services/source/payload/iris/iris_service.c

```diff
@@ -177,12 +177,20 @@ SAT_returnState iris_service_app(csp_packet_t *packet) {
          * 2. Transfer image
          */
         uint32_t image_length;
+        uint16_t image_count;
+        char filename[100];
```

Comment:
You can use a REDCONF_NAME_MAX (or something similar, I forget what it is for sure) instead of a guess here. Makes the code slightly more portable